finetune/target_models/masked_language_model.py

```diff
@@ -96,6 +98,34 @@ def __init__(self, *args, **kwargs):
     def _get_input_pipeline(self):
         return MaskedLanguageModelPipeline(self.config)
 
+    def predict_top_k_report(self, input_text, k=5, context=None, **kwargs):
+        prediction_info = self._inference(
```

Comment:
I'm pretty sure this would break if chunk_long_sequences is turned on. Worth checking and documenting/raising an error, as it is possible the error results could be really weird (functional but offset) if the extra tokens are added to MLM.

Reply:
AFAICT that is correct, would be nice if there was a way of checking that without passing through any config? I don't need the rest of it.

Reply:
Got it!